sphinxcontrib/confluencebuilder/publisher.py

```diff
@@ -205,8 +205,8 @@ def getDescendents(self, page_id):
             if page_id:
                 search_fields = {'cql': 'ancestor=' + str(page_id)}
             else:
-                search_fields = {'cql': 'space=' + self.space_name +
-                    ' and type=page'}
+                search_fields = {'cql': 'space="' + self.space_name +
```

Comment:
what about if the space name contains a double quote?

Reply:
At any point where `getDescendents` is invoked, the configured space name has already been validated [existence check; and space name character requirement includes `a-Z0-9` or (personal spaces only) `~`].

Since this is the only CQL call in this extension, I do not see a major need to attempt to make a complete CQL escape helper call. That being said, if you are strongly for making it "perfect", I can try to make a proper CQL statement builder utility class.